sonar-scanner-engine/src/main/java/org/sonar/scanner/scan/ModuleScanContainer.java

```diff
@@ -92,6 +93,8 @@ protected void doBeforeStart() {
   private void addCoreComponents() {
     add(
       module.definition(),
+      // still injected by some plugins
+      new Project(module.definition()),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "Project"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)